### DISCUSSION TITLE
Garmin Varia UT800 - how to change intensity of the light

### DISCUSSION THREAD

**Discussion by kregielpl** [maca88/SmartBikeLights#67] [General]
Hi,

Recently, I bought a Garmin Varia UT800 and did my first ride yesterday. I set the default mode to day flash, but this mode is too strong. I'm wondering if it is possible to lower the intensity of that mode using SBL?

Thanks for your help!

**Comment by maca88** [comment]
Hi,

yes, it is possible to set to whatever mode you like by using the Smart Bike Lights Lights Configurator: https://maca88.github.io/SmartBikeLights/

**Comment by kregielpl** [comment]
But I would like to use the Day flash mode because it lasts 25 hours, but the intensity of the strobe light is too strong for other riders.
